apps/api/src/modules/user/user.repository.ts

```diff
@@ -103,4 +115,97 @@ export class UserRepository {
     }
     return respondUpdatedUser
   }
+
+  async getActorsWorkHistory(paramId: number): Promise<GetJobCardDto[]> {
+    const jobs = await this.prisma.job.findMany({
+      orderBy: {
+        jobId: Prisma.SortOrder.desc,
+      },
+      where: {
+        status: JobStatus.FINISHED,
+        Application: {
+          some: {
+            actorId: paramId,
+            status: ApplicationStatus.OFFER_ACCEPTED,
+          },
+        },
+      },
```

Comment:
Maybe can filter refund by NOT: {refundStatus: RefundStatus. ACCEPTED} but optional